tests/components/zha/test_light.py

```diff
@@ -63,6 +67,43 @@
 }
 
 
+@asynctest.mock.patch(
```

Comment:
The diff will be smaller in the future, when we reach Python 3.8 as minimum version and can replace asynctest with std lib again, if we import `patch` from `asynctest`.

Reply:
Will we address this later in a future PR?

Reply:
Yes. I’d like to get this PR into the beta release